include/grpc/impl/codegen/port_platform.h

```diff
@@ -82,36 +82,38 @@
    things.  */
 
 #if !defined(GPR_NO_AUTODETECT_PLATFORM)
+#if defined(_WIN64) || defined(WIN64) || defined(_WIN32) || defined(WIN32)
```

Comment:
Not really ? The following line is a specialization to do the detection 32/64 bits, whereas this one is to detect if we're compiling for Windows at all.